intelmq/conf/harmonization.conf

```diff
@@ -311,6 +311,10 @@
             "description": "The URL of a given abuse feed, where applicable",
             "type": "URL"
         },
+        "feed.accuracy": {
```

Comment:
@sebix you are right... If you check the message.py it already has that field being passed on to the events, but I forgot to put it into the harmonization document.